libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -38,6 +38,7 @@ export type AccordionItemBaseProps = {
   onClick?: () => void
   onBlur?: () => void
   onFocus?: () => void
+  boxProps?: BoxProps
```

Comment:
I don't think Accordion Items should be given the full API of the Box component

Reply:
I agree. I will change this.